app/liquid_tags/stackblitz_tag.rb

```diff
@@ -17,44 +21,42 @@ def render(_context)
       partial: PARTIAL,
       locals: {
         id: @id,
-        view: @view,
-        file: @file,
+        params: @params,
         height: @height
       },
     )
   end
 
   private
 
-  def valid_id?(id)
-    id =~ ID_REGEXP
-  end
+  def parsed_input(input)
+    id, *params = input.split
+    match = pattern_match_for(id, REGEXP_OPTIONS)
+    raise StandardError, I18n.t("liquid_tags.stackblitz_tag.invalid_stackblitz_id") unless match
 
-  def parse_id(input)
-    input_no_space = input.split.first
-    raise StandardError, I18n.t("liquid_tags.stackblitz_tag.invalid_stackblitz_id") unless valid_id?(input_no_space)
+    return [match[:id], nil] unless params_present?(match) || params
 
-    input_no_space
+    build_link_with_params(match[:id], (params_present?(match) || params))
   end
 
-  def parse_input(input, validator)
-    inputs = input.split
+  def params_present?(match)
+    return unless match.names.include?("params")
 
-    # Validation
-    validated_views = inputs.filter_map { |input_option| validator.call(input_option) }
-    raise StandardError, I18n.t("liquid_tags.stackblitz_tag.invalid_options") unless validated_views.length.between?(0,
-                                                                                                                     1)
-
-    validated_views.length.zero? ? "" : validated_views.join.to_s
+    match[:params].delete("?")
   end
 
-  def valid_view?(option)
-    option.match(VIEW_OPTION_REGEXP)
+  def build_link_with_params(id, params)
+    params = params.split("&") if params.is_a?(String)
+    vetted_params = params.filter_map { |param| param if valid_param(param) }.join("&")
```

Comment:
Same as in the Stackery PR

```suggestion
    vetted_params = params.select { |param| valid_param(param) }.join("&")
```